api/management/commands/create_groups.py

```diff
@@ -0,0 +1,31 @@
+from collections.abc import Iterable
+from typing import Any
+
+from django.core.management.base import BaseCommand, CommandError, CommandParser
+
+from api.models import Teacher
+from api.processors.auxil.algorithms import GroupBuilderAlgorithm
+
+
+class Command(BaseCommand):
+    """
+    Triggers automatic group creation.
+    If teacher_ids are not specified, triggers for all available teachers.
+    """
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument("--teacher_ids", nargs="*", type=int)
```

Comment:
Isn't it more correct to call it `teacherS_ids`? I always do so 🤔 
@lemontree210

Reply:
Well, according to English grammar, `teacher_ids` is correct in this case. But we don't need to impose this on a project level I don't think 😄